pkg/cmd/cli/describe/chaindescriber.go

```diff
@@ -92,6 +93,15 @@ func (d *ChainDescriber) Describe(ist *imageapi.ImageStreamTag, includeInputImag
 		return "", NotFoundErr(fmt.Sprintf("%q", ist.Name))
 	}
 
+	markers := buildanalysis.FindCircularBuilds(g, osgraph.DefaultNamer)
+	if len(markers) > 0 {
+		for _, marker := range markers {
+			if strings.Contains(marker.Message, ist.Name) {
+				return fmt.Sprintf("Exiting examination of %s. %s", ist.Name, marker.Message), nil
```

Comment:
On the returning of error, I interpreted the build-chain path as only returning errors when there was a problem in processing data (bad format, etc.).  In this case, the data was correctly analyzed.  It simply found a user error - a misconfiguration.

On the contents of the message, I went back and forth on whether to just return the marker.Message, but thought that somehow repeating the IST they passed in would provide clarity.  But I'm definitely on the fence with that.  Suggestions welcome, but based on @bparees 's feedback I'm minimally thinking of just returning marker.Message, but will wait on additional feedback.

With that, adding the marker.Message to the normal chain-build output (nested is/bc paris, with each is and bc on a separate line, and each line having increased tabbing) definitely did not flow / look valid in my opinion, and is in fact repetitive, as the marker.Message will display the circular chain.